src/components/TransactionResult.tsx

```diff
@@ -0,0 +1,53 @@
+import {
+  Alert,
+  AlertDescription,
+  AlertIcon,
+  AlertTitle,
+  Flex,
+} from "@chakra-ui/react";
+
+export enum ActionName {
+  Deposit = "Deposit",
+  Withdraw = "Withdraw",
+  Approve = "Approve",
+  Associate = "Associate",
+  Mint = "Mint",
+  Claim = "Claim",
+}
+
+const getAlertTitle = (isSuccess = true, actionType?: ActionName) => {
+  return `${actionType} ${isSuccess ? "success" : "error"}`;
+};
```

Comment:
we can drop this and just show things we need right away
`<AlertTitle>${actionName} success</AlertTitle>`
`<AlertTitle>${actionName} error</AlertTitle>`